pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -247,16 +250,48 @@ func (o *Options) Run() error {
 					break
 				}
 			}
-			if update == nil {
+
+			// Check if the --to version present in condiional update
+			for _, upgrade := range cv.Status.ConditionalUpdates {
+				if c := findCondition(upgrade.Conditions, "Recommended"); c != nil && c.Status != metav1.ConditionTrue {
+					if upgrade.Release.Version == o.To {
+						conditionalUpdate = &configv1.Update{
+							Version: upgrade.Release.Version,
+							Image:   upgrade.Release.Image,
+						}
+						if o.AllowNotRecommended {
+							fmt.Fprintf(o.ErrOut, "warning: with --allow-not-recommended you have accepted the risks with %s and bypassing %s=%s %s: %s\n", o.To, c.Type, c.Status, c.Reason, c.Message)
```

Comment:
nit: "you have accepted ... and bypassing ..." should be "... bypassed..." to match "accepted".

Reply:
fixed with https://github.com/openshift/oc/pull/986/commits/55b2321b9028f17aa2475cf75c650727b6539ab7